docs/data-collection.md

```diff
@@ -81,6 +81,11 @@ For the OpenShift 4 Developer Preview we will be sending back these exact attrib
   // cluster:node_instance_type_count:sum is the number of nodes
   // of each instance type and role.
   '{__name__="cluster:node_instance_type_count:sum"}',
+  //
+  // kubevirt metrics to be collected
```

Comment:
Remove this, I don't think there is anything special about kubevirt.  However, you need to describe in more detail in your description why you want this.  Also make sure it is a complete sentence(s) (punctuation).

Reply:
Ack, will either drop the comment or add a more informative one.